ui/create_restore_page.go

```diff
@@ -323,64 +307,222 @@ func (pg *createRestore) mainContent(gtx layout.Context) layout.Dimensions {
 	)
 }
 
-func (pg *createRestore) restore(gtx layout.Context) layout.Dimensions {
-	dims := layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-		layout.Rigid(func(gtx C) D {
-			return layout.W.Layout(gtx, func(gtx C) D {
-				return pg.hideRestoreWallet.Layout(gtx)
+func (pg *createRestore) restore(gtx layout.Context, common pageCommon) layout.Dimensions {
+	op.TransformOp{}.Add(gtx.Ops)
+	paint.Fill(gtx.Ops, pg.theme.Color.LightGray)
+	dims := layout.Stack{Alignment: layout.S}.Layout(gtx,
+		layout.Expanded(func(gtx C) D {
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					return layout.Inset{Left: values.MarginPadding20}.Layout(gtx, func(gtx C) D {
+						return layout.W.Layout(gtx, func(gtx C) D {
+							return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+								layout.Rigid(func(gtx C) D {
+									return pg.hideRestoreWallet.Layout(gtx)
+								}),
+								layout.Rigid(func(gtx C) D {
+									return layout.Inset{Top: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
+										return pg.theme.H6("Restore wallet").Layout(gtx)
+									})
+								}),
+							)
+						})
+					})
+				}),
+				layout.Rigid(func(gtx C) D {
+					return common.UniformPadding(gtx, func(gtx C) D {
```

Comment:
Currently common.UniformPadding() uses a uniform inset of  24. This should be the reason you have too much padding between you page title and the seed phrase section which should be 16 Dp.

My recommendation is to manually define the Inset values instead of using common.UniformPadding().
```suggestion
m := values.MarginPadding24
		return layout.Inset{
		        Top: values.MarginPadding24,
			Left: m,
			Right: m,
			Bottom: m,
		}.Layout(gtx, func(gtx C) D {
```

<img width="532" alt="image" src="https://user-images.githubusercontent.com/27733432/116500358-50e2a400-a8a6-11eb-88b5-d73bb7289d9a.png">

too much padding. it should be 16. Kindly
<img width="473" alt="image" src="https://user-images.githubusercontent.com/27733432/116500416-6f489f80-a8a6-11eb-9be1-59cf840db65d.png">

Reply:
> Currently common.UniformPadding() uses a uniform inset of 24. This should be the reason you have too much padding between you page title and the seed phrase section which should be 16 Dp.
> 
> My recommendation is to manually define the Inset values instead of using common.UniformPadding().
> 
> <img alt="image" width="532" src="https://user-images.githubusercontent.com/27733432/116500358-50e2a400-a8a6-11eb-88b5-d73bb7289d9a.png">
> 
> too much padding. it should be 16. Kindly
> <img alt="image" width="473" src="https://user-images.githubusercontent.com/27733432/116500416-6f489f80-a8a6-11eb-9be1-59cf840db65d.png">

The Container type can be used. It is a wrapper for the inset type that was added to differentiate when an inset is used as a margin or padding in the code.

Reply:
The Container type can be used. It is a wrapper for the inset type that was added to differentiate when an inset is used as a margin or padding in the code.